docs/source/examples/examples.myst

```diff
@@ -4,12 +4,26 @@
 ---
 maxdepth: 2
 ---
-ibmq-backends.rst
-hamiltonians.rst
-simple_landscape.rst
+ibmq-backends.myst
+hamiltonians.md
+simple_landscape.myst
 maxcut-demo.myst
 pyquil_demo.myst
 cdr_api.md
 mitiq-paper/mitiq-paper-codeblocks
 pec-tutorial.myst
 ```
+
+## Links to the examples for My Binder
+
+[Error mitigation on IBMQ backends](ibmq-backends.myst)
+
+[Defining Hamiltonians as Linear Combinations of Pauli Strings](hamiltonians.md)
+
+[Mitigating the energy landscape of a variational circuit with Mitiq](simple_landscape.myst)
+
+[Solving MaxCut with QAOA improved with Mitiq](maxcut-demo.myst)
+
+[CLifford data regression API](cdr_api.md)
+
+[Mitiq paper codeblocks](mitiq-paper/mitiq-paper-codeblocks.md)
```

Comment:
It's really not great that the TOC has to be duplicated. Is there any other way to do this?